test/utils/fieldUtils.js

```diff
@@ -52,14 +53,29 @@ class FieldDataTypes {
    * Enter a text into the CaseLink field
    * @returns CCDStringField Object
    */
-  async enterIntoCaseLinkField(value, id){
+  async enterIntoCaseLinkField(value, id) {
     let css = await FIELDS.CASE_LINK.cssTag;
     let type = await FIELDS.CASE_LINK.type;
     let field = await new CCDStringField(css, type, id);
     await field.enterText(value);
     return field;
   }
 
+  /**
+   * Sets the value into the AddressLine1 field of the address field specified by id.
+   * If no id is specified it will populate AddressLine1 of the first found address.
+   * @param value
+   * @param id
+   * @returns CCDAddressUKField Object
+   */
+  async enterIntoAddressLine1Field(value, id) {
```

Comment:
This works for now but I wonder in the future if this should be refactored to just 'enter into addressUK field' and should pass an object or an array which would then be resolved in the CCDAddressUKField class and map tot he fields in the complexType

Reply:
Yes, I have left the rest of the fields for later.